tests/test-splinter.yaml

```diff
@@ -26,10 +26,18 @@ services:
       - ../:/project/splinter
     command: |
         bash -c "
+            echo \"Running workspace tests...\" && \
             cargo test && \
-            echo \"Running optional feature tests...\" && \
-            (cd libsplinter && cargo test --features zmq-transport) &&
-            cargo test signing::sawtooth --manifest-path libsplinter/Cargo.toml \
-                --features \"sawtooth-signing-compat\"
+            echo \"Running features=default tests...\" && \
+            cargo test \
```

Comment:
This will run all the tests, even if they are not affected by the feature (all splinterd tests, when we are only changing libsplinter for example) Is this desirable?

Reply:
I was actually wrong about this, ignore